### PR TITLE
【Hackathon No.106】Add paddle model and infer with InfiniTensor

### PR DESCRIPTION
**Pull request by learner2468**
## What do we do in this pull request?

1. Add paddle model training and evaluation for Resnet, Densenet and Inception model
2. Convert paddle model to onnx model
3. Infer with InfiniTensor and calculate accuracy

## Test evidence

```
~/workplace/infinitensor-test-scripts/paddle$ python test_densenet.py
import backend: <module 'backend' from '/home/libaoming/workplace/InfiniTensor/pyinfinitensor/src/pyinfinitensor/backend.cpython-310-x86_64-linux-gnu.so'>
W1201 18:44:33.782874 1914783 gpu_resources.cc:119] Please NOTE: device: 0, GPU Compute Capability: 8.0, Driver API Version: 12.2, Runtime API Version: 11.2
W1201 18:44:33.783813 1914783 gpu_resources.cc:149] device: 0, cuDNN Version: 8.7.
The loss value printed in the log is the current step, and the metric is the average value of previous steps.
Epoch 1/5
step 782/782 [==============================] - loss: 1.2599 - acc: 0.5014 - 135ms/step
Epoch 2/5
step 782/782 [==============================] - loss: 1.1066 - acc: 0.6896 - 135ms/step
Epoch 3/5
step 782/782 [==============================] - loss: 0.8127 - acc: 0.7784 - 135ms/step
Epoch 4/5
step 782/782 [==============================] - loss: 0.7291 - acc: 0.8272 - 134ms/step
Epoch 5/5
step 782/782 [==============================] - loss: 0.5695 - acc: 0.8592 - 135ms/step
Eval begin...
step 157/157 [==============================] - loss: 0.8962 - acc: 0.8294 - 67ms/step
Eval samples: 10000
I1201 18:53:34.326586 1914783 program_interpreter.cc:185] New Executor is Running.
2023-12-01 18:53:34 [INFO]      Static PaddlePaddle model saved in onnx.save/paddle_model_static_onnx_temp_dir.
[Paddle2ONNX] Start to parse PaddlePaddle model...
[Paddle2ONNX] Model file path: onnx.save/paddle_model_static_onnx_temp_dir/model.pdmodel
[Paddle2ONNX] Paramters file path: onnx.save/paddle_model_static_onnx_temp_dir/model.pdiparams
[Paddle2ONNX] Start to parsing Paddle model...
[Paddle2ONNX] Use opset_version = 11 for ONNX export.
[Paddle2ONNX] PaddlePaddle model is exported as ONNX format now.
2023-12-01 18:53:34 [INFO]      ONNX model saved in onnx.save/densenet.onnx.
test acc: 0.8286
```

### CODE REVIEW

File: examples/python/paddle_densenet.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
+
+import paddle
+import numpy as np
```
Comment (kilinchange) [inline examples/python/paddle_densenet.py:3]: np 在代码里并没有用到，这条 import 语句可以去掉。

File: examples/python/paddle_inception.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
+import paddle
+import numpy as np
```
Comment (kilinchange) [inline examples/python/paddle_inception.py:2]: remove np

File: examples/python/paddle_resnet.py [hunk 3]
```diff
@@ -0,0 +1,82 @@
+
+import paddle
+import numpy as np
```
Comment (kilinchange) [inline examples/python/paddle_resnet.py:3]: remove np

File: examples/python/paddle_densenet.py (hunk 1 above)
Comment (learner2468) [reply examples/python/paddle_densenet.py]: Thanks, done

File: examples/python/paddle_inception.py (hunk 2 above)
Comment (learner2468) [reply examples/python/paddle_inception.py]: Thanks, done

File: examples/python/paddle_resnet.py (hunk 3 above)
Comment (learner2468) [reply examples/python/paddle_resnet.py]: Thanks, done

**Review by kilinchange — APPROVED** [review]

**Review by wanghailu0717 — APPROVED** [review]

**Pull request merged**